scripts/publish-module.gradle

```diff
@@ -0,0 +1,58 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+group = PUBLISH_GROUP_ID
+version = PUBLISH_VERSION
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+
+                groupId PUBLISH_GROUP_ID
+                artifactId PUBLISH_ARTIFACT_ID
+                version PUBLISH_VERSION
+
+                // Two artifacts, the `aar` (or `jar`) and the sources
+                if (project.plugins.findPlugin("com.android.library")) {
+                    from components.release
+                } else {
+                    from components.java
+                }
+
+                pom {
+                    name = PUBLISH_ARTIFACT_ID
+                    description = 'Glia Android Widgets SDK'
+                    url = 'https://www.glia.com'
```

Comment:
NIT: I think it would be nice to extract the description, URL and company name in the same style as `PUBLISH_GROUP_ID`, `PUBLISH_ARTIFACT_ID`.

Reply:
fixed